Sources/Build/BuildOperation.swift

```diff
@@ -45,10 +45,12 @@ public final class BuildOperation: PackageStructureDelegate, SPMBuildCore.BuildS
 
     /// The loaded package graph.
     private var packageGraph: PackageGraph?
-
     /// The stdout stream for the build delegate.
     let stdoutStream: OutputByteStream
 
+    /// Wether the build is in a verbose mode.
+    private let isVerbose: Bool
```

Comment:
wdyt about storing `Verbosity` instead of `Bool` as the member, and adding a "isVerbose" computed property that is calculated off of that enum value?

Reply:
it's akin to what's in `ToolWorkspaceDelegate`, would you prefer to change there as well?